docs/source/recipes.rst

```diff
@@ -437,3 +437,38 @@ different environment.
 
    To test operations with a small interactive job, a 'test' group can be used
    to ensure that the operations do not try to run on multiple cores or GPUs.
+
+Pass options to operations run through containers or another environment
+========================================================================
+
+When executing an operation in a container (e.g. Singularity or Docker) or a different environment
```

Comment:
Motivating by formatting like problem-solution. I'm not sure my explanation is right though
```suggestion
When executing an operation in a container (e.g. Singularity or Docker) or a different environment, the operation will not receive command line flags from the submitting process.
```